pom.xml

```diff
@@ -6,7 +6,7 @@
 
     <groupId>com.acrobot.chestshop</groupId>
     <artifactId>chestshop</artifactId>
-    <version>3.12.3-SNAPSHOT</version>
+    <version>3.12.4-SNAPSHOT</version>
```

Comment:
Please don't adjust the version, that will only be done when 3.12.3 was actually released.

Reply:
Changed - will let you mark as resolved on evaluation